validator/client/beacon-api/event_handler.go

```diff
@@ -48,28 +50,29 @@ func (h *EventHandler) subscribe(sub eventSub) {
 	h.Unlock()
 }
 
-func (h *EventHandler) get(ctx context.Context, topics []string, eventErrCh chan<- error) error {
+func (h *EventHandler) get(ctx context.Context, topics []string) error {
 	if len(topics) == 0 {
 		return errors.New("no topics provided")
 	}
 
-	allTopics := strings.Join(topics, ",")
-	log.Info("Starting listening to Beacon API events on topics " + allTopics)
-	url := h.host + "/eth/v1/events?topics=" + allTopics
-	req, err := http.NewRequestWithContext(ctx, http.MethodGet, url, nil)
-	if err != nil {
-		return errors.Wrap(err, "failed to create HTTP request")
-	}
-
-	req.Header.Set("Accept", api.EventStreamMediaType)
-	req.Header.Set("Connection", "keep-alive")
-
-	resp, err := h.httpClient.Do(req)
-	if err != nil {
-		return errors.Wrap(err, "failed to perform HTTP request")
-	}
-
 	go func() {
+		h.running = true
```

Comment:
hmm is this the best approach to this/ is this safe?

Reply:
It can be improved, I think. I will push a new commit